syft/frameworks/crypten/context.py

```diff
@@ -135,27 +131,21 @@ def wrapper(*args, **kwargs):
 
             rank_to_worker_id = dict(zip(range(0, len(workers)), [worker.id for worker in workers]))
 
-            # TODO: run ttp in a specified worker
-            # if crypten.mpc.ttp_required():
-            #     ttp_process, _ = _new_party(
-            #         crypten.mpc.provider.TTPServer,
-            #         world_size,
-            #         world_size,
-            #         master_addr,
-            #         master_port,
-            #         (),
-            #         {},
-            #     )
-            #     ttp_process.start()
-
```

Comment:
I think this TODO still need to be implemented, maybe we should keep it here

Reply:
Done